ZuluContent/Spells/Base/SpellHelper.cs

```diff
@@ -253,12 +258,12 @@ public static int GetDamageAfterResist(Mobile caster, Mobile target, double dama
             if (damage < 1)
                 damage = 1;
 
-            damage = (int) (damage * (1.0 + evalInt - resist) / 200.0);
+            damage = (int) (damage * (1.0 + (evalInt - resist) / 200.0));
 
             // Inverting the efficiency bonus, e.g. Mages get less spell damage, warriors get more
-            var temp = damage;
-            target.FireHook(h => h.OnModifyWithMagicEfficiency(target, ref temp));
-            damage -= damage - temp;
+            // var temp = damage;
```

Comment:
Just wondering - for what reason is this still commented?

Reply:
Yeah probably need to make a note of this as a balancing issue; it basically makes resist extremely OP for mages, and mage vs mage PVP everything does almost 0 damage making it just a waste of time. It's actually disabled in the ZHA5 scripts as well for that reason I think.